pkg/utils/utils.go

```diff
@@ -125,6 +125,7 @@ func DiscoverSriovDevices(withUnsupported bool) ([]sriovnetworkv1.InterfaceExt,
 		if dputils.IsSriovPF(device.Address) {
 			iface.TotalVfs = dputils.GetSriovVFcapacity(device.Address)
 			iface.NumVfs = dputils.GetVFconfigured(device.Address)
+			iface.NUMANode = dputils.GetDevNode(device.Address)
```

Comment:
Do we need to set NUMANode attribute even it's not specified or error occured https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin/blob/b3aea760a5e2d92414d73f9195c731a532282dad/pkg/utils/utils.go#L211?

Reply:
-1 means that it failed to get it, even though it might still be on another numa node then 0. This just exposes whatever the underlying lib reports. I'm not entirely sure why we are exposing this info.